doc/README_Source.md

```diff
@@ -163,10 +162,11 @@ The following settings are used to configure the Cosmos DB Kafka Source Connecto
 | connect.cosmosdb.connection.endpoint | uri | the endpoint for the Cosmos DB Account | Required |
 | connect.cosmosdb.containers.topicmap | string | comma separeted topic to collection mapping, eg. topic1#coll1,topic2#coll2 | Required |
 | connect.cosmosdb.containers | string | list of collections to monitor |  Required |
+| connect.cosmosdb.setMessageKey | boolean | set if the Kafka message key should be set to Cosmos DB document ID | Required |
```

Comment:
i would add a 2nd dependant config property here, which when setMessageKey is true, you can choose the property you want to use as the Key. just an xpath property, default it to "id", but at least let the user choose to set the property to something else if they wish. 

new dependent property name could be messageKeyPath. defaulted to "id".